docs/tutorials/overwrite-player-at-runtime.md

```diff
@@ -0,0 +1,154 @@
+# What does overwrite player at runtime mean ? And why may you need it
+
+We have a plugins and custom build tutorial section, but sometimes you may only want
+to change just a bit html/css/js to the player,
+custom build may add unecessary complex to your application and deployment process.
+
+Luckily, we can do that another way (overwrite shaka-player at runtime),
+in this tutorial, we'll guide you through a few simple examples to get a feel how to do it.
+
+Of course, there are pros and cons to each approach, let talk about pros:
+- Reuse current CDN (you can still use cdnjs, jsdivr, google hosted libraries, etc..).
+- Use some commits are merged upstream (master) but not yet in released version.
+- Overwrite default inline css shaka-player.
+
+In this tutorial, we will guide you to: add a custom button, modify css style,
+modify a function, detach/add new event listener to an existing element,
+(in practice you could even rewrite shaka-player a whole at runtime).
+
+#### Add a button
+When using shaka player from CDN, your app will have access to global `shaka` variable
+(make sure you don't use global variable `shaka` anywhere in your app,
+we could accidentally overwrite each other).
+
+First let take a look at these button (don't be afraid):
+- [Cast](https://github.com/google/shaka-player/blob/master/ui/cast_button.js)
+- [Fast forward](https://github.com/google/shaka-player/blob/master/ui/fast_forward_button.js)
+- [Play](https://github.com/google/shaka-player/blob/master/ui/play_button.js)
+- [Air play](https://github.com/google/shaka-player/blob/master/ui/airplay_button.js)
+
+What do you see in common ?
+A structure compose of 3 parts: a button extends of basic [element](https://github.com/google/shaka-player/blob/master/ui/element.js) attached to shaka.ui namespace
+A Factory class and a register method for that Factory class.
+
+Let copy that and put it right after fullscreen button!
+
+```js
+shaka.ui.SkipButton = class extends shaka.ui.Element {
+  constructor(parent, controls) {
+    super(parent, controls);
+
+    // The actual button that will be displayed
+    this.button_ = document.createElement('button');
+    this.button_.textContent = 'Skip current video';
+    // Screen readers will read "skip button"
+    this.button_.setAttribute('aria-label', 'Skip');
+    this.parent.appendChild(this.button_);
+  }
+};
+
+shaka.ui.SkipButton.Factory = class {
+  create(rootElement, controls) {
+    return new shaka.ui.SkipButton(rootElement, controls);
+  }
+};
+
+shaka.ui.Controls.registerElement(
+    'skip', new shaka.ui.SkipButton.Factory());
+```
+
+That basically it! When you init shaka-player, make sure you call that `skip` button out
+
+```js
+// init config shaka-player section
+config['controlPanelElements'] = [
+  'play_pause', 'mute', 'volume', 'time_and_duration', 'spacer',
+  'overflow_menu', 'fullscreen', 'skip' // order matter, we put skip behind fullscreen
+]
+```
+
+#### Modify html
```

Comment:
This is not really a good idea.  If you want to change the position of the seek bar, you should file a bug and we can add a configuration for this.  Changing HTML elements like this can break layouts and alignments.